src/macfc-security-hub-sync.ts

```diff
@@ -441,6 +645,7 @@ export class SecurityHubJiraSync {
         )
       }
     } catch (e: unknown) {
+      console.log(JSON.stringify(finding))
```

Comment:
Is this console.log necessary when it is not in debug mode?

Reply:
removed